<ui version="4.0" >
 <class>SettingsProxyForm</class>
 <widget class="QWidget" name="SettingsProxyForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>239</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="4" row="0" column="0" >
    <widget class="QListWidget" name="listProxys" />
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="pushAdd" >
     <property name="text" >
      <string>Add</string>
     </property>
     <property name="icon" >
      <iconset>../../</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="pushEdit" >
     <property name="text" >
      <string>Edit</string>
     </property>
     <property name="icon" >
      <iconset>../../</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="pushDelete" >
     <property name="text" >
      <string>Delete</string>
     </property>
     <property name="icon" >
      <iconset>../../</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
